pkg/util/binding.go

```diff
@@ -97,3 +98,15 @@ func MergeTargetClusters(oldCluster, newCluster []workv1alpha2.TargetCluster) []
 	}
 	return newCluster
 }
+
+// RescheduleTriggeredAfterLastSchedule judge whether rescheduleTriggeredAt is later than lastScheduledTime.
+func RescheduleTriggeredAfterLastSchedule(rescheduleTriggeredAt, lastScheduledTime *metav1.Time) bool {
```

Comment:
How about renaming it to `needTriggerScheduling`?

Reply:
I update it to `RescheduleRequired`